include/mbedtls/aes.h

```diff
@@ -67,6 +67,49 @@
 /* MBEDTLS_ERR_AES_HW_ACCEL_FAILED is deprecated and should not be used. */
 #define MBEDTLS_ERR_AES_HW_ACCEL_FAILED                   -0x0025  /**< AES hardware accelerator failed. */
 
+#if defined( MBEDTLS_CHECK_PARAMS )
+
+#if defined(MBEDTLS_PARAM_FAILED_CALLBACK)
+#define MBEDTLS_AES_VALIDATE_RET( cond )                                \
```

Comment:
Because this will replicate in all modules, I wonder if we can make a macro accepting the module name, here `AES`, as a parameter, and outputs the code block. I.e. `MBEDTLS_VALIDATE_RET( cond, module_name )`, and then `#define MBEDTLS_AES_VALIDATE_RET( cond ) MBEDTLS_VALIDATE_RET( cond, AES )`?